### ISSUE TITLE
can't send via arcor

### ISSUE THREAD

**Issue by GoogleCodeExporter** [satyajit319/websmsdroid#91]
```
What steps will reproduce the problem?
1. TrYing to send sms using arcor connector
2. But sending fails
3. Error message: arcor: kein absender angegeben

But the caller-id is definitly entered in the preference.






What version of the product are you using? On what operating system?
Acer liquid with 1.6 websms3

Please provide any additional information below.


```

Original issue reported on code.google.com by `nolife...@googlemail.com` on 12 Mar 2010 at 6:58

**Comment by GoogleCodeExporter** [comment]
```
@nolifedie

Can you tell as please where did you entered your username, etc?

Which fields did you fill / checked?
Main Prefs -> Your mobile number ?
Arcor Prefs -> Use validated number?
Arcor Prefs -> User default sender?
Arcor Prefs -> Custom sender?


Does update to get free sms count work?


```

Original comment by `herrl...@gmail.com` on 12 Mar 2010 at 9:53

**Comment by GoogleCodeExporter** [comment]
```
Ok, I can reproduce this on my device.
```

Original comment by `herrl...@gmail.com` on 12 Mar 2010 at 9:58

**Comment by GoogleCodeExporter** [comment]
```
I'm working at this problem. At least sending now ist workin, wenn I choose "Use
Validated number". Wenn I diselect this option  "kein absender angegeben" 
arises :(
At this moment no ide why. But I do not give up :)

```

Original comment by `herrl...@gmail.com` on 13 Mar 2010 at 11:43
- Changed state: **Started**

**Comment by GoogleCodeExporter** [comment]
```
OK now seems to be fixed. Unfortunately no more customSender  is possible. So
either you should chosse "Validierte Nummer" or your 'username'@arcor.de ist 
reported
as sender. Felix will upload the test version soon.

P.S.
"Validierte Nummer" ist the better way, you get 160 letters + you number as 
sender.
```

Original comment by `herrl...@gmail.com` on 15 Mar 2010 at 7:58

**Comment by GoogleCodeExporter** [comment]
```
this should be fixed with arcor connector 1.2, thx to everyone :)
```

Original comment by `felix.bechstein` on 15 Mar 2010 at 10:51
- Changed state: **Fixed**
